fitting_source/1D_fitting/plot_4_11B_Quad_NMR.py

```diff
@@ -44,6 +42,23 @@
 plt.tight_layout()
 plt.show()
 
+# %%
+# Estimate noise statistics from the dataset
+coords = experiment.dimensions[0].coordinates
+noise_region = np.where(np.logical_and(coords < -75e-6, coords > -100e-6))
+noise_data = experiment[noise_region]
+
```

Comment:
The noise area grabbed in this example only has a slight upward slope due to the nearby sideband.

Reply:
Can you update the docs. Thanks.